supplier_app/models.py

```diff
@@ -74,7 +84,13 @@ class TaxPayer(models.Model):
         blank=True,
         null=True,
         verbose_name=_("Comments"),
-        )
+    )
+    eb_entity = models.ManyToManyField(
```

Comment:
This `ManyToManyField` is still necessary?

Reply:
Yes, this is an error